include/utils/VtlibUtils.php

```diff
@@ -647,8 +647,10 @@ function vtlib_purifyForSql($string, $skipEmpty=true) {
 }
 
 function getvtlib_open_popup_window_function($popupmodule,$fldname,$basemodule) {
+    if(file_exists('modules/'.$popupmodule.'/'.$popupmodule.'.php')){
 	include_once 'modules/'.$popupmodule.'/'.$popupmodule.'.php';
 	$mod = new $popupmodule();
+    }
 	if (method_exists($mod, 'getvtlib_open_popup_window_function')) {
```

Comment:
Shouldn't this "if" be inside the "file_exists if"?  If you don't load the file $mod will be undefined and the method_exists will emit a warning at least.
Please, move the "method_exists if" inside the "file_exists if"